README.md

```diff
@@ -4,28 +4,47 @@
 [![Build Status](https://travis-ci.org/pingcap/tikv.svg?branch=master)](https://travis-ci.org/pingcap/tikv)
 
 
-TiKV is inspired by the design of Google Spanner and HBase, but much simpler (Doesn't depend on any distributed file system).
+TiKV is a Distributed Key-Value Database which mainly refers to the design of Google Spanner and HBase, but much simpler (Don't depend on any distributed file system). And the implementation of TiKV/TiDB is a more convenient and more understandable replacement for Spanner/F1. We've implemented the Raft consensus algorithm in Rust and stored consensus state in RocksDB. It not only guarantees consistency for data but also makes use of placement driver to implement sharding (split && merge) and data migration automatically. The transaction model is similar to Google's Percolator, and with some performance improvements. In fact, we provide snapshot isolation (SI) and serializable snapshot isolation (SSI), and then externally consistent reads and writes in distributed transactions. Some details can see [TiKVserver software stack](#TiKVserver-software-stack). And some primary features list as follow.
```

Comment:
Remove "And the implementation of TiKV/TiDB is a more convenient and more understandable replacement for Spanner/F1". Because we can't do real comparison currently(But i suppose Spanner and F1 should be much more powerful than TiKV and TiDB).